### PR TITLE
217 operation datastore functions

### LINKED ISSUE #217: Operation datastore functions
- [x] assembly_script_get_op_keys
- [x] assembly_script_has_op_key
- [x] assembly_script_get_op_data

### PR DESCRIPTION
**Pull request by 0xSwapFeeder**
This PR aims to resolve #217.

### CODE REVIEW

**Review by Elli610 — COMMENTED** [review]
it looks good

File: assembly/__tests__/env-opdatastore.spec.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+describe('Testing mocked Operation Datastore', () => {
+  test('hasOpKey - key not exists', () => {
+    expect(env.hasOpKey(stringToBytes('key1'))).toBeNull();
+  });
+
+  test('getOpKey - key exists', () => {
+    env.set(stringToBytes('key1'), stringToBytes('value1'));
+    expect(env.hasOpKey(stringToBytes('key1'))).not.toBeNull();
+    resetStorage();
```
Comment (Ben-Rey) [inline assembly/__tests__/env-opdatastore.spec.ts:14]: you have a beforeEach function in as-pect. Maybe you could use it for resetStorage

Comment (0xSwapFeeder) [reply assembly/__tests__/env-opdatastore.spec.ts]: Nice! Didn't know about that, actually done using 'afterEach' to be sure Storage is reset after this test group.

Comment (Ben-Rey) [reply assembly/__tests__/env-opdatastore.spec.ts]: perfect

**Review by Ben-Rey — APPROVED** [review]

**Pull request merged**